audio-worklet/bitcrusher.html

```diff
@@ -56,9 +55,8 @@ <h1 id="title">Bit Crusher</h1>
       let gain = 0.5;
       let controller = new BitcrusherDemo(bitDepth, reduction, 0.5);
```

Comment:
Don't you mean `new BitCrusherDemo(bitDepth, reduction, gain)`?  Otherwise, what's the difference between the two 0.5 values?